/*
Copyright 2020 The Magma Authors.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

syntax = "proto3";

import "orc8r/protos/common.proto";

package magma.lte;

option go_package = "magma/lte/cloud/go/protos";

service S1apService {
    // Returns state of the S1 connected eNBs
    rpc GetENBState (magma.orc8r.Void) returns (EnbStateResult) {}
}

// enb_state_map { eNB IDs -> # UEs connected }
message EnbStateResult {
    map<uint32, uint32> enb_state_map = 1;
}
